frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -1149,6 +1217,11 @@ type ClusterServiceVersionStateProps = {
   activeNamespace?: string;
 };
 
+type AlertToCreateCustomResourceProps = {
+  obj: ClusterServiceVersionKind;
+  cr: any;
```

Comment:
It would be good to have a type definition for what we expect the annotation to be.

Reply:
Is there a generic kind I can use?

Reply:
> Is there a generic kind I can use?

Is there a design somewhere that describes the annotation?

It looks like we're treating it as an array in the code, but the issue says, "The "CR_TEMPLATE" will be a complete YAML template for the CR." It looks like the implementation doesn't match the epic description.